packages/core/package.json

```diff
@@ -32,6 +32,7 @@
     "!**/*.spec.js.map"
   ],
   "dependencies": {
+    "@chainsafe/libp2p-noise": "^4.1.1",
```

Comment:
Why are we using this version (from chain safe) now?

Reply:
They are now maintaining the project

Reply:
ok, good to know